app/src/main/java/io/pslab/activity/LuxMeterActivity.java

```diff
@@ -94,11 +99,39 @@ protected void onCreate(Bundle savedInstanceState) {
         tvShadow.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                if (bottomSheetBehavior.getState() == BottomSheetBehavior.STATE_EXPANDED)
+                if(bottomSheetBehavior.getState()==BottomSheetBehavior.STATE_EXPANDED)
                     bottomSheetBehavior.setState(BottomSheetBehavior.STATE_HIDDEN);
                 tvShadow.setVisibility(View.GONE);
             }
         });
+
+        bottomNavigationView.setOnNavigationItemSelectedListener
```

Comment:
We removed this bottom navigation bar from Lux meter and @Avjeet has implemented a settings section for it using menu bar. Please remove these references.